pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -52,6 +52,10 @@ type InterPodAffinityArgs struct {
 	// HardPodAffinityWeight is the scoring weight for existing pods with a
 	// matching hard affinity to the incoming pod.
 	HardPodAffinityWeight int32
+
+	// IgnoreExistingInterPodAffinities configures the scheduler to ignore existing pods' affinity
+	// rules when scoring candidate nodes, unless the incoming pod has inter-pod affinities.
+	IgnoreExistingInterPodAffinities bool
```

Comment:
The name should indicate that this is only related to preferred terms; so perhaps we can name it `IgnorePreferredTermsOfExistingPods`

Reply:
Makes sense - done.